test/generate_ssl_tests.pl

```diff
@@ -43,6 +43,7 @@ sub print_templates {
     # Add the implicit base configuration.
     foreach my $test (@ssltests::tests) {
         $test->{"server"} = { (%ssltests::base_server, %{$test->{"server"}}) };
+	$test->{"server2"} = { (%ssltests::base_server, %{$test->{"server2"}}) };
```

Comment:
I originally had that, but then due to the template, server2 wasn't being put into the .conf files. I'll try to work through that.